src/main/resources/static/common/js/demoAdmin.js

```diff
@@ -413,6 +430,9 @@ mod.controller('demo_admin_controller', function($rootScope, $scope, $http, $fil
         }).then(function(result) {
             localStorage.setItem("status", "true");
             localStorage.setItem("save_succeeded", $filter('translate')('SAVING_SUCCEEDS'));
+            if (result.data.data.mqType !== "KAFKA") {
```

Comment:
Please wait for [story_pda1092](https://github.com/parasoft/parasoft-demo-app/pull/210) to merge first. 
This variable name may be changed. 
Also you might have made duplicate changes as the other pull request.